.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      - experimental_v1.0
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v1
+        with:
+          auto_fix: true
+          black: true
+          flake8: true
+          flake8_args: "--config /Users/asablayrolles/code/utils/flake8_config.ini"
+
```

Comment:
How about running isort as well? 
There is [an action for it](https://github.com/isort/isort-action)

```yaml
- name: Run isort
  uses: isort/isort-action@master
  with:
    requirementsFiles: "requirements.txt dev_requirements.txt"
```